src/views/attachment.ts

```diff
@@ -0,0 +1,161 @@
+/*!
+ * © 2025 Atypon Systems LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { ManuscriptNode } from '@manuscripts/transform'
+
+import { fileMainDocumentIcon } from '../icons'
+import { FileAttachment } from '../lib/files'
+import { Trackable } from '../types'
+import BlockView from './block_view'
+import { createNodeView } from './creators'
+
+// Extended file type that includes properties from the article editor
+interface ExtendedFileAttachment extends FileAttachment {
+  type?: string
+}
+
+export class AttachmentView extends BlockView<Trackable<ManuscriptNode>> {
+  private container: HTMLElement
+
+  public createElement = () => {
+    this.createContainer()
+    this.renderView()
+  }
+
+  public updateContents() {
+    this.renderView()
+  }
+
+  private renderView() {
+    this.container.innerHTML = ''
+    this.container.classList.remove('attachment-hidden')
+
+    const file = this.getFileFromAttachment()
+    if (!file) {
+      this.container.classList.add('attachment-hidden')
+      return
+    }
+
+    if (this.isPDF(file)) {
+      this.createPDFPreview(file)
+    } else {
+      this.container.classList.add('attachment-hidden')
+    }
+  }
+
+  private createContainer() {
+    this.container = document.createElement('div')
+    this.container.classList.add('attachment-item')
+    this.container.addEventListener('click', (e) => {
+      e.stopPropagation()
+      this.setMainDocumentSelection()
+    })
```

Comment:
we can navigate to the view by selecting it, like we do with [inline files](https://github.com/Atypon-OpenSource/manuscripts-article-editor/blob/3cb5adb4b40dbdbef765429a544988c50932b4fb/src/components/FileManager/InlineFilesSection.tsx#L147)